packages/core/src/server/utils/__tests__/initial-state.tests.ts

```diff
@@ -87,4 +87,30 @@ describe("initialState", () => {
     const url = new URL("https://site.com/page/2?some=query#some-hash");
     expect(initialState({ settings, url }).frontity.platform).toBe("server");
   });
+
+  it("should set the frontity.options correctly", () => {
+    const url = new URL(
+      "https://site.com/page/2?frontity_name=mySite&frontity_source_auth=hello"
+    );
+
+    // The frontity_name & frontity_source_auth should be present in the options
+    expect(initialState({ settings, url }).frontity.options)
+      .toMatchInlineSnapshot(`
+      Object {
+        "frontity_name": "mySite",
+        "frontity_source_auth": "hello",
```

Comment:
Hmm, I don't have a strong opinion because on one hand camelCase == consistency across frontity packages. 

On the other, keeping them in snake_case perhaps makes it more clear that they were part of the URL? So this way, they are consistent across the URL and the options object.

I lean towards agreeing with Luis here.

Reply:
I absolutely agree with you on both points. I don't have a strong opinion either, I just felt we should ask ourselves this question before making this public.

@darerodz, what do you think?

Reply:
Yeah, well, I would choose camelCase mainly because I'm more used to it. :camel: 

Some JavaScript libraries already transform `kebab-case` or `snake_case` into `camelCase` (like what React does with component properties) so I think it would not be hard to understand for developers why the options are in `snake_case` in the URL and in `camelCase` in JavaScript/TypeScript.

Reply:
okay, I ll transform it to 🐫Case in that case 🙂